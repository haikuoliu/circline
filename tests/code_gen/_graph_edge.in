node a = node("a");
node b = node("b");
node c = node("c");
node d = node("d");
node e = node("e");

/* Basic Type */
graph gh = a -> 2&b -> 1.2&c -> (1>3)&d -> (1<2)&e -> "Hello"&a;

print( int(gh@(a,b)) );
print( float(gh@(b,c)) );
print( bool(gh@(c,d)) );
print( bool(gh@(d,e)) );
print( string(gh@(e,a)) );
