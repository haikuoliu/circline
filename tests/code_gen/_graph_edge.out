2
1.200000
false
true
Hello
